org.sil.FieldWorks.yml

```diff
@@ -0,0 +1,787 @@
+            tee --append Mercurial/mercurial.ini <<< \
+              "fixutf8=/app/lib/flexbridge/MercurialExtensions/fixutf8/fixutf8.py"
+            # Help hg find python (in /app).
+            perl -pi -e 's#/usr/bin/python#/usr/bin/env python#' Mercurial/hg
+
+            make install-core
+        sources:
+          - type: git
+            url: https://github.com/sillsdev/flexbridge.git
+            # Branch develop
+            commit: 7dbc684697944d59fad7f091f5266f99d1a289a9
+          - type: file
+            # To update: Choose a new l10n artifact at
+            # https://github.com/sillsdev/flexbridge/releases , possibly after
+            # running the localizations GHA.
+            url: https://github.com/sillsdev/flexbridge/releases/download/l10n-v3.3.0-38-gf6ef2ba8/FieldWorksL10ns.zip
+            sha512: d395f26be1ef1777351e5515e34a3f7004d126590b00049aa5ec03d7ab392af60a26bce31a6af16a9eefdb5e79854f8bf24d40a61aed214558b3f35041b53149
+            dest: l10n/
+          - type: file
+            url: https://dist.nuget.org/win-x86-commandline/v5.9.0/nuget.exe
+            sha512: 980b62387ce9540eff0d49ef6dd5b4da121a8648646a465bc6445534e9ec0d7605483fad933b26f9b0b3797db4d4b1ec82ed354b3daf13e1dcc0818292815dca
+            dest: build/
+          # These nuget packages are installed by build/FLExBridge.proj by
+          # directly running nuget.exe. To update: Check on what current
+          # packages and versions are needed with
+          # `grep NuGetCommand build/FLExBridge.proj`. Then manually edit the
+          # yml to change or add.
+          # Note that attempting to install
+          # SIL.Windows.Forms.GeckoBrowserAdapter results in it trying to
+          # identify dependencies to install, and works differently than the
+          # other nuget installations that use a similar command. A solution is
+          # to write the GeckoBrowserAdapter nuget file to dest nuget-sources,
+          # rather than nuget-specific. The dependencies may not need special
+          # handling, but if they start to, then note that the dependencies
+          # don't seem to be mentioned in the .nupkg file itself, nor on the
+          # nuget.org web site, but can be determined by running in flexbridge
+          # `msbuild build/FLExBridge.proj -t:RestoreBuildTasks` and observing
+          # how it resolves dependencies.
+          - flexbridge-sources-nuget-specific.yml
+          # Use local nuget packages
+          - type: shell
+            commands:
+              - |
+                tee nuget.config <<< '<?xml version="1.0" encoding="utf-8"?>
+                <configuration>
+                  <packageSources>
+                    <add key="LocalNugetPackages" value="'$(pwd)/nuget-sources'" />
+                  </packageSources>
+                  <disabledPackageSources>
+                    <add key="nuget.org" value="true" />
+                  </disabledPackageSources>
+                </configuration>'
```

Comment:
you could use a "type: inline" to have this as an inline source.

Reply:
Thank you. I was not aware of inline.